WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -119,6 +133,21 @@ - (void)dealloc
     [[UIApplication sharedApplication] removeObserver:self forKeyPath:WPApplicationIconBadgeNumberKeyPath];
 }
 
+#pragma mark - UIViewControllerRestoration methods
+
+- (void)encodeRestorableStateWithCoder:(NSCoder *)coder
+{
+    [coder encodeInteger:self.selectedIndex forKey:WPTabBarSelectedIndexKey];
+    [super encodeRestorableStateWithCoder:coder];
+}
+
+- (void)decodeRestorableStateWithCoder:(NSCoder *)coder
+{
+    // Set last selected tab bar item
```

Comment:
ie: if more info needs to be added it should be placed in the documentation comments for property `selectedIndex`.